user-data/vault.sh.tftpl

```diff
@@ -49,3 +49,30 @@ else
 		--tls-cert-file "$VAULT_TLS_CERT_FILE" \
 		--tls-key-file "$VAULT_TLS_KEY_FILE"
 fi
+
+# Only create the script if pushgateway_urls is not empty
+if [ -n "${pushgateway_urls}" ]; then
+
+  # Create Vault health check script
+  cat << 'SCRIPT' > /usr/local/bin/vault_health_push.sh
+  #!/bin/bash
+
+  # Get AWS instance ID
+  INSTANCE_ID=$(ec2metadata --instance-id)
+  INSTANCE_IP=$(ec2metadata --local-ipv4)
+
+  # Check Vault health status
+  VAULT_STATUS=$(/usr/bin/curl -s https://127.0.0.1:8200/v1/sys/health | /usr/bin/jq -r '.sealed' | /bin/sed 's/true/1/' | /bin/sed 's/false/0/')
+
+  # Loop through each Pushgateway URL and send metrics
+  for PUSHGATEWAY_URL in ${pushgateway_urls}; do
+    echo "# HELP vault_sealed The state of Vault (0 = unsealed, 1 = sealed)\n # TYPE vault_sealed gauge\n vault_sealed{instance=\"$INSTANCE_IP\", instance_id=\"$INSTANCE_ID\"} $${VAULT_STATUS}" | /usr/bin/curl --data-binary @- $${PUSHGATEWAY_URL}/metrics/job/vault_health/instance/$${INSTANCE_IP}
+  done
+SCRIPT
+
+  chmod +x /usr/local/bin/vault_health_push.sh
+
+  # Add a system cron job to run every minute
+  echo "*/5 * * * * root /usr/local/bin/vault_health_push.sh >> /var/log/push_vault_metrics.log 2>&1" > /etc/cron.d/pushmetrics
```

Comment:
Minor: either adjust the comment to "every five minutes" or the crontab back to one minute.

Why the change from one minute to five minutes?